pkg/k8s/network_policy.go

```diff
@@ -136,6 +145,16 @@ func ParseNetworkPolicy(np *networkingv1.NetworkPolicy) (api.Rules, error) {
 		egresses = append(egresses, egress)
 	}
 
+	// Isolate namespace on ingress
```

Comment:
In order for me to understand this comment in 12 months:
```
// Convert the k8s default-deny model to the Cilium default-deny model
// spec:
//   policyTypes:
//   - ingress
```

Reply:
Done